cdapython/constant_variables.py

```diff
@@ -12,7 +12,7 @@ class Constants:
     _VERSION: str = "2023.5.4"
     _DATABASETABLE_VERSION: str = "all_merged_subjects_v3_2_final"
```

Comment:
_DATABASETABLE_VERSION and _DATABASETABLE should come out in the production release